package.json

```diff
@@ -52,7 +52,7 @@
     }
   ],
   "dependencies": {
-    "@particle/device-constants": "^2.0.0",
+    "@particle/device-constants": "^3.1.6",
     "binary-version-reader": "^1.1.1",
```

Comment:
You'll need to bump binary-version-reader and particle-usb to keep the versions consistent

Reply:
thanks @monkbroc! A couple of thoughts from my side:
1. I will bump up the node/npm versions for the particle-cli. The `binary-version-reader : 1.1.1` still worked with node v12 and npm v6. The more latest versions of BVR need [upgraded versions of node and npm](https://github.com/particle-iot/binary-version-reader/commit/0aa18f2181da9ddf4b4ebaa102a216b6d418f1e9) 

2. Is this the correct order? Add necessary changes to BVR and Particle-USB that supports Orson and then continue with this PR.